zio-http/src/test/scala/zio/http/ZClientAspectSpec.scala

```diff
@@ -45,6 +48,30 @@ object ZClientAspectSpec extends ZIOSpecDefault {
           output.head.endsWith("ms\n"),
         )
       },
+      test("proxy") {
```

Comment:
Take a look at `ClientProxySpec` for how to test this. I think you can probably just add a test there.